src/nucleic_acid/k80/absolute.jl

```diff
@@ -41,8 +41,8 @@ end
   end
   α = mod.α; β = mod.β
 
-  e₁ = 0.25 * exp(-4 * β * t)
-  e₂ = 0.5 * exp(-2 * (α + β) * t)
+  e₁ = 0.25 * exp(-β * t)
```

Comment:
Is this change meant to be left in? Or was there an error with how K80 model P matrices were specified?

Reply:
Not an error, but they were already set up to do scaling. This change makes them act the same as the other models, only scale with `P(mod, true)`.